packages/shopping/src/services/email.service.ts

```diff
@@ -0,0 +1,41 @@
+import {bind, BindingScope} from '@loopback/core';
+const nodeMailer = require('nodemailer');
```

Comment:
Perhaps use `import nodemailer from 'nodemailer';`

Reply:
Types for nodemailer are not mature yet. Can be updated in future

Reply:
Even `https://www.npmjs.com/package/@types/nodemailer` not up to scratch?

Reply:
You can propose usage and will take it from there. At the moment this is lower priority given the core functionality is incomplete

Reply:
@dougal83 see https://github.com/nodemailer/nodemailer/issues/1094. Literally changing to import breaks the whole package setup hence my argument above

Reply:
Ah, great thank you. So @types need an update. I looked at my old project and see that nodemailer was in sync with types at the time. Might see if I can contribute update.

Reply:
Types do not require update.  Error message was not clear.